packages/mpp/mpp.0.3.4/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "mpp"
+version: "0.3.4"
+maintainer: "Philippe Wang <philippe.wang@gmail.com>"
+authors: [ "Philippe Wang <philippe.wang@gmail.com>" ]
+license: "ISC"
+homepage: "https://github.com/ocaml/MPP-language-blender"
+dev-repo: "git+https://github.com/ocaml/MPP-language-blender.git"
+bug-reports: "https://github.com/ocaml/MPP-language-blender/issues"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "4.03"}
```

Comment:
The package does not build on 4.02, I made the bound stricter:
```

#=== ERROR while compiling mpp.0.3.4 ==========================================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://github.com/ocaml/MPP-language-blender/archive/0.3.4.tar.gz)
# path                 ~/.opam/4.02/.opam-switch/build/mpp.0.3.4
# command              ~/.opam/4.02/bin/dune build -p mpp -j 72
# exit-code            1
# env-file             ~/.opam/log/mpp-23-ae662f.env
# output-file          ~/.opam/log/mpp-23-ae662f.out
### output ###
#     ocamldep src/.mpp.objs/mpp_init.ml.d (exit 2)
# (cd _build/default && /home/opam/.opam/4.02/bin/ocamldep.opt -modules -impl src/mpp_init.ml) > _build/default/src/.mpp.objs/mpp_init.ml.d
# File "src/mpp_init.ml", line 30, characters 0-86:
# Error: Syntax error: attributes not expected.
#     ocamldep src/.mpp.objs/mpp_charstream.ml.d (exit 2)
# (cd _build/default && /home/opam/.opam/4.02/bin/ocamldep.opt -modules -impl src/mpp_charstream.ml) > _build/default/src/.mpp.objs/mpp_charstream.ml.d
# File "src/mpp_charstream.ml", line 319, characters 0-1005:
# Error: Syntax error: attributes not expected.

-     ocamldep src/.mpp.objs/mpp_init.ml.d (exit 2)
- (cd _build/default && /home/opam/.opam/4.02/bin/ocamldep.opt -modules -impl src/mpp_init.ml) > _build/default/src/.mpp.objs/mpp_init.ml.d
- File "src/mpp_init.ml", line 30, characters 0-86:
- Error: Syntax error: attributes not expected.
-     ocamldep src/.mpp.objs/mpp_charstream.ml.d (exit 2)
- (cd _build/default && /home/opam/.opam/4.02/bin/ocamldep.opt -modules -impl src/mpp_charstream.ml) > _build/default/src/.mpp.objs/mpp_charstream.ml.d
- File "src/mpp_charstream.ml", line 319, characters 0-1005:
- Error: Syntax error: attributes not expected.
```

Reply:
OK, thanks! 

(I feel I should change MPP to support 4.02 but at the same time I don't think there are users for MPP on 4.02 so I'll likely leave it unchanged.)